static/images/2020-12-rust-survey-2020/language-interop.svg

```diff
@@ -0,0 +1 @@
7.55Q493.72186 37.003124 493.64374 36.26875ZM493.4875 32.346874Q492.53436 32.7375 490.6125 33.003124Q489.53436 33.159374 489.08124 33.3625Q488.64374 33.55 488.39374 33.925Q488.14374 34.3 488.14374 34.76875Q488.14374 35.471874 488.675 35.940624Q489.20624 36.409374 490.2375 36.409374Q491.2531 36.409374 492.03436 35.971874Q492.83124 35.51875 493.20624 34.753124Q493.4875 34.14375 493.4875 32.9875L493.4875 32.346874ZM497.55 38.409374L499.2531 38.659374Q499.3625 39.45625 499.84686 39.815624Q500.5031 40.3 501.64374 40.3Q502.8625 40.3 503.51874 39.815624Q504.1906 39.33125 504.425 38.45625Q504.5656 37.909374 504.55 36.190624Q503.39374 37.55 501.675 37.55Q499.53436 37.55 498.3625 36.003124Q497.1906 34.45625 497.1906 32.3Q497.1906 30.815624 497.72186 29.565624Q498.26874 28.315624 499.28436 27.628124Q500.3156 26.940624 501.6906 26.940624Q503.51874 26.940624 504.72186 28.425L504.72186 27.175L506.33124 27.175L506.33124 36.14375Q506.33124 38.565624 505.83124 39.565624Q505.34686 40.58125 504.26874 41.159374Q503.20624 41.753124 501.64374 41.753124Q499.8 41.753124 498.64374 40.925Q497.5031 40.096874 497.55 38.409374ZM499.0031 32.175Q499.0031 34.221874 499.8156 35.159374Q500.6281 36.096874 501.84686 36.096874Q503.05 36.096874 503.8625 35.159374Q504.6906 34.221874 504.6906 32.2375Q504.6906 30.33125 503.84686 29.3625Q503.0031 28.39375 501.8156 28.39375Q500.64374 28.39375 499.8156 29.346874Q499.0031 30.3 499.0031 32.175ZM515.97186 34.20625L517.78436 34.440624Q517.3625 36.01875 516.1906 36.909374Q515.03436 37.784374 513.22186 37.784374Q510.95624 37.784374 509.6125 36.39375Q508.28436 34.9875 508.28436 32.45625Q508.28436 29.83125 509.6281 28.39375Q510.97186 26.940624 513.1281 26.940624Q515.20624 26.940624 516.51874 28.3625Q517.84686 29.76875 517.84686 32.346874Q517.84686 32.503124 517.83124 32.815624L510.09686 32.815624Q510.1906 34.51875 511.0656 35.425Q511.9406 36.33125 513.2375 36.33125Q514.20624 36.33125 514.8781 35.83125Q515.5656 35.315624 515.97186 34.20625ZM510.1906 31.3625L515.9875 31.3625Q515.8781 30.065624 515.33124 29.409374Q514.4875 28.39375 513.14374 28.39375Q511.9406 28.39375 511.1125 29.20625Q510.28436 30.003124 510.1906 31.3625ZM523.15936 34.01875Q523.14374 33.675 523.14374 33.503124Q523.14374 32.45625 523.4406 31.70625Q523.65936 31.14375 524.14374 30.565624Q524.4875 30.14375 525.39374 29.346874Q526.3156 28.534374 526.58124 28.065624Q526.8625 27.58125 526.8625 27.01875Q526.8625 25.9875 526.05 25.221874Q525.2531 24.440624 524.09686 24.440624Q522.97186 24.440624 522.22186 25.14375Q521.47186 25.846874 521.2375 27.33125L519.425 27.1125Q519.675 25.1125 520.8781 24.05Q522.08124 22.9875 524.0656 22.9875Q526.175 22.9875 527.425 24.128124Q528.675 25.26875 528.675 26.89375Q528.675 27.83125 528.22186 28.628124Q527.78436 29.409374 526.5031 30.55Q525.64374 31.315624 525.3781 31.675Q525.1281 32.034374 524.9875 32.503124Q524.8625 32.971874 524.84686 34.01875L523.15936 34.01875ZM523.05 37.55L523.05 35.55L525.05 35.55L525.05 37.55L523.05 37.55Z" fill-rule="nonzero"/></svg>
```

Comment:
Can the pie charts please be 2d instead of 3d. In 3d the slices at the top of the pie chart look smaller relative to a same percentage at the bottom due to perspective.

Reply:
Done! 👍